public/app/features/explore/Explore.tsx

```diff
@@ -614,28 +609,26 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
               <div className={styles.exploreContainer}>
                 {datasourceInstance ? (
                   <>
-                    <ContentOutlineItem title="Queries" icon="arrow">
```

Comment:
I think keeping this would be the solution for differentiating between nested queries and the query panel container. We could even have a prop that we can pass here to the `ContentOutlineItem` and tell it if this is a parent or a child. Or we could do these dynamically but traversing the DOM and looking if an item is a child of another item. 

Maybe using props and declaring it would make it easier to understand how things connected.

Reply:
Decided to go with the option of having a 'level' prop which if not set is automatically a parent ('root'), while children need to be set explicitly